src/extension.ts

```diff
@@ -2450,7 +2450,7 @@ async function setup(context: vscode.ExtensionContext, progress?: ProgressHandle
 
     return { getApi: (version: api.Version) => {
         // Since our API is backwards compatible, we can make our version number match that which was requested.
-        if (version === api.Version.v1 || version === api.Version.v2) {
+        if (version === api.Version.v1 || version === api.Version.v2 || version === api.Version.v3) {
             ext.api.version = version;
```

Comment:
not sure what this piece of code does, but curious why there isn't v4 here even though api.ts has 
    version: api.Version = api.Version.v4;

Reply:
Good catch.

Reply:
Fixed.